tests/nested/core/core20-set-efi-boot-vars/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: Check that EFI boot variables are successfully on UC20
+
+details: |
+    This test checks that EFI boot variables are correctly set during
+    installation.  In particular, there should be a Boot#### variable with
+    data pointing to either \EFI\ubuntu\shimx64.efi or \EFI\boot\bootx64.efi,
+    depending on the version of the pc-gadget snap, and the first number in the
+    BootOrder variable should be the corresponding number.
+
+systems: [ubuntu-20.04-64]
```

Comment:
Should run for 22 as well (or, better, for 20+)

Reply:
I don't want it to run on 23.04, for example, so I've put it back to 20.04-64 and 22.04-64. I think this can be manually updated when core24 is added to tests.